bin/setup

```diff
@@ -10,3 +10,4 @@ go get github.com/Masterminds/glide \
 gometalinter --install
 bin/build
 bundle install
+npm install
```

Comment:
Why npm over yarn?

Reply:
NPM 5 has all the features that make Yarn cool, and it is installed by default and doesn't require another setup step here and on dev machines.